From 2e131d0a0dbfaa16139ba7e508454cde01b6cc1e Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 26 Jan 2024 18:09:31 +0100
Subject: [PATCH 0585/1695] dts: stm32: vdd_sdcard not always-on for
 stm32mp257f-dk

Remove always-on on vdd_sdcard power supplies in order
to reduce power consumption.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 42a9338936e2172c7e53ea2adabdfed6627a9849 dts: stm32: vdd_sdcard not always-on for stm32mp257f-dk]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I0d171647ed7817d64fff0668e18eccfc971709c0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/354904
Tested-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index b87d0556c..c93cb2073 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -279,7 +279,6 @@
 				regulator-name = "vdd_sdcard";
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
-				regulator-always-on;
 				st,regulator-bypass-microvolt = <3300000>;
 				st,pwrctrl_sel = <3>;
 				st,pwrctrl_reset;
-- 
2.47.2

